### PR TITLE
Adds rules to pmd check

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Adds following rules:
clone.xml
finalizers.xml	     
imports.xml		     		
logging-java.xml
braces.xml

clone.xml, finalizers.xml and brances.xml didn't raise any problems
The problems raised by the two rules - imports.xml and loggin-java.xml have been fixed in separate commits. 

#### What has been done to verify that this works as intended?
No change in the functionality

#### Why is this the best possible solution? Were any other approaches considered?
It makes the code more readable by removing the unnecessary Full Qualified name

#### Are there any risks to merging this code? If so, what are they?
No

### CODE REVIEW

File: collect_app/src/test/java/org/odk/collect/android/preferences/AdminPreferencesActivityTest.java [hunk 1]
```diff
@@ -42,7 +44,7 @@ public void shouldUpdateAdminSharedPreferences() throws NullPointerException {
         for (String adminKey : AdminKeys.ALL_KEYS) {
             Preference preference = adminPreferencesFragment.findPreference(adminKey);
             if (preference instanceof CheckBoxPreference) {
-                System.out.println("Testing " + adminKey);
+                Timber.d("Testing " + adminKey);
```
Comment (yurilaguta) [inline collect_app/src/test/java/org/odk/collect/android/preferences/AdminPreferencesActivityTest.java]: some other code checker will complain that it doesn't use String template. Timber.d("Testing %s", adminKey);

**Comment by Yurii-Laguta** [comment]
Looks great! Only minor comment with regard to Timber using string concatenation.

**Review by yurilaguta — APPROVED** [review]
Looks great! Only minor non-blocking comment with regard to Timber using string concatenation.

**Review by lognaturel — APPROVED** [review]
Thanks @shobhitagarwal1612 and thanks for the good review, @Yurii-Laguta!

**Pull request merged**